README.md

```diff
@@ -28,6 +28,15 @@ Questions, bug reporting and issue tracking are provided by GitHub. Please
 report all bugs by creating a new issue with the bug tag. You can ask
 questions by creating a new issue with the question tag.
 
+Contributing
+------------
+
+Contributing to DTK is easy. Just send us a [pull request](https://help.github.com/articles/using-pull-requests/).
```

Comment:
If there are no test with a code, it's really hard to say that the code is correct but more importantly it is impossible to be sure that this piece of code stays correct over time.